### PR TITLE
Modify width of ui container, fine tune css for settings pages and org header

### LINKED ISSUE #24302: Org private label's location will be changed if you access between projects and members page
### Description

Maybe the reason is that the width of `class="org-visibility"` is different:
in projects page:
image
in member page:
image

They are using different tmpl file, but have same code. Maybe the root problem in other places.

Can also reproduce this issue in gitea.com 

### Gitea Version

1.20.0+dev-555-gc9c8a96c4

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

_No response_

### How are you running Gitea?

build

### Database

None

### PR DESCRIPTION
**Pull request by HesterG**
Close #24302 
Part of #24229, Follows #24246
This PR focused on css style fine tune, main changes:

1. Give `.ui.ui.ui.container` a width of `1280px` with a max-width of `calc(100vw - 64px)`, so the main contents looks better on large devices. 

    Before (screen width: 1920px):

    Screen Shot 2023-04-25 at 10 14 55

    Screen Shot 2023-04-25 at 10 14 28

    
    After (screen width: 1920px):

    Screen Shot 2023-04-25 at 09 46 38

    Screen Shot 2023-04-25 at 09 46 48

2. Share styles for table elements in all levels settings pages to fix overflow of runners table on mobile and for consistency (The headers on mobile can be further improved, but haven't found a proper way yet).

    Before:
  
    Screen Shot 2023-04-25 at 10 16 43Screen Shot 2023-04-25 at 10 16 15

    After:

     Screen Shot 2023-04-25 at 10 28 35

3. Use [stackable grid](https://fomantic-ui.com/collections/grid.html#stackable) and [device column width](https://fomantic-ui.com/examples/responsive.html) for responsiveness for some pages (repo/org collaborators settings pages, org teams related page)

    Before:
    
    Screen Shot 2023-04-25 at 10 20 30Screen Shot 2023-04-25 at 10 19 34Screen Shot 2023-04-25 at 10 22 16

    After:
    

    Screen Shot 2023-04-25 at 09 59 11Screen Shot 2023-04-25 at 09 59 01Screen Shot 2023-04-25 at 10 05 10

4. Fixed #24302 by sharing label related css in reporg.css
    After:
    
    https://user-images.githubusercontent.com/17645053/234161424-1560603d-aaa6-4330-be0b-14bdab31e763.mov

5. Fine tune repo tags settings page
    Before:
    Screen Shot 2023-04-25 at 10 24 57

    After:
    Screen Shot 2023-04-25 at 10 08 48

### CODE REVIEW

File: web_src/css/shared/settingtable.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+/* This css is for table elements inside admin, organization, and repository settings 
+  TODO: Right now there is no table inside user settings, but in the future if there is table
+  in user settings, should also use this common style here.
+*/
+.admin .table.segment,
+.repository.settings .table.segment,
+.organization.settings .table.segment {
```
Comment (wxiaoguang) [inline web_src/css/shared/settingtable.css]: I think we can introduce a general `<div class="twelve wide column g-grid-setting-right">` for every "layout_head", then we only need to write styles for `g-grid-setting-right`

File: web_src/css/shared/settingtable.css [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  padding: 0;
+  font-size: 13px;
```
Comment (wxiaoguang) [inline web_src/css/shared/settingtable.css]: use `em` instead?

File: web_src/css/shared/settingtable.css (hunk 1 above)
Comment (HesterG) [reply web_src/css/shared/settingtable.css]: Good idea, I will add this class

**Review by lunny — APPROVED** [review]

File: web_src/css/shared/settingstable.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  padding-bottom: 5px;
+}
+
+.g-grid-setting-right .table.segment:not(.select) th:first-of-type,
+.g-grid-setting-right .table.segment:not(.select) td:first-of-type {
+  padding-left: 15px !important;
+}
+
+.g-grid-setting-right .table.segment form tbody button[type="submit"] {
+  padding: 5px 8px;
```
Comment (wxiaoguang) [inline web_src/css/shared/settingstable.css]: Why this? There are also other `<button>` elements in forms

Comment (HesterG) [reply web_src/css/shared/settingstable.css]: This is copied from the old code, which is table related.

Comment (wxiaoguang) [reply web_src/css/shared/settingstable.css]: I am pretty sure that we do not need them ... will do more fine tunes

**Comment by wxiaoguang** [comment]
Made more changes

Before:

image

image

image

After:

image

image

image

**Comment by wxiaoguang** [comment]
It seems that you didn't resolve merge conflicts correctly, many pages got 500 error ....... ea0f79ae3250e50f2eb76547ba066a474692ad36

Please review and resolve conflicts carefully.

**Review by wxiaoguang — APPROVED** [review]
I have tested many pages, LGTM

File: web_src/css/admin.css [hunk 4]
```diff
@@ -4,30 +4,6 @@
   margin: 12px -1rem -1rem;
 }
 
-.admin .table.segment {
-  padding: 0;
-  font-size: 13px;
-  overflow-x: auto;
```
Comment (HesterG) [inline web_src/css/admin.css:10]: `overflow-x` is needed, might need to open another PR to fix this:

Screen Shot 2023-04-27 at 09 01 24
Screen Shot 2023-04-27 at 09 09 54

Comment (wxiaoguang) [reply web_src/css/admin.css:10]: Good catch, luckily we only need one `overflow-x` now. Maybe we need to write a comment about how to test such style (I just forgot ...)

Comment (wxiaoguang) [reply web_src/css/admin.css:10]: Fix admin table layout #24370

**Pull request merged**